app/controllers/application_controller.rb

```diff
@@ -25,17 +25,15 @@ class ApplicationController < ActionController::Base
     :should_present_taxonomy_navigation?,
   )
 
-  def current_tasklist_ab_test
-    GovukNavigationHelpers::CurrentTasklistAbTest.new(
-      current_tasklist: current_tasklist,
-      request: request
-    )
+  def current_step_nav
+    @step_nav = GovukNavigationHelpers::StepNavContent.current_step_nav(request.path)
   end
-  helper_method :current_tasklist_ab_test
+  helper_method :current_step_nav
 
-  def tasklist_content
-    current_tasklist if current_tasklist && current_tasklist.is_page_included_in_ab_test?
+  def show_step_nav?
+    current_step_nav && current_step_nav.show_step_nav?
   end
+  helper_method :show_step_nav?
```

Comment:
More of a question than a comment: what's this line do? Only line 31 looks very similar, is it adding both :current_step_nav and :show_step_nav to the helper or something?

Reply:
That's exactly what this is :-) .  It makes the methods available to views.